### ISSUE TITLE
Test Down?

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#1802]
I can't get Test to open. Is it down?

**Comment by sharpphyl** [comment]
Is that why we can't get a new user checked in?

**Comment by dustymc** [comment]
If you're trying to do that at test, then yes.

**Comment by sharpphyl** [comment]
No, not at test, but when she changes  her password, she doesn't get
confirmation that it's acceptable and I can't give her any access to our
collection.

**Comment by dustymc** [comment]
Details please - username, error, screenshots, etc. are all likely to be useful.

**Comment by sharpphyl** [comment]
Details:  agent:  Brenna O'Neill, logon: bo'neill_dmns.

She's left for the day so I can't send a screen-shot of what she got when
she tried to validate as a user.  She'll be back Tuesday, so we'll try
again, but maybe it's just a burp.

Phyllis

**Comment by dustymc** [comment]
>bo'neill_dmns.

http://handbook.arctosdb.org/documentation/users.html

>account name must use only letters and numbers (ASCII characters A-Z, a-z, and 0-9), and cannot begin with a number.

**Comment by sharpphyl** [comment]
Aargh!  Operator error.  I'll change her account name and try again.
Thanks!
